src/opera/pge/base/base_pge.py

```diff
@@ -300,8 +300,8 @@ def _create_catalog_metadata(self):
             'PGE_Name': self.runconfig.pge_name,
             'PGE_Version': self.PGE_VERSION,
             'SAS_Version': self.SAS_VERSION,
-            'Input_Files': self.runconfig.get_input_filenames(),
-            'Ancillary_Files': self.runconfig.get_ancillary_filenames(),
+            'Input_Files': list(map(basename, self.runconfig.get_input_filenames())),
+            'Ancillary_Files': list(map(basename, self.runconfig.get_ancillary_filenames())),
```

Comment:
Nice:  I assume this obviates the need to use 'basename' later.

Reply:
Nice fix 👍